webcompat/webhooks/helpers.py

```diff
@@ -39,6 +49,36 @@ def extract_browser_label(body):
         return None
 
 
+def extract_extra_label(metadata_dict):
+    """Return the 'extra' label from metadata_dict"""
+    extra_label = metadata_dict.get('extra_label', None)
+    if extra_label:
+        extra_label = extra_label.lower()
+        return extra_label.encode('utf-8')
+    else:
+        return None
+
```

Comment:
Why the `else`? the line before the `if`, `None is already assigned`

```python
def extract_extra_label(metadata_dict):
    """Return the 'extra' label from metadata_dict"""
    extra_label = metadata_dict.get('extra_label', None)
    if extra_label:
        extra_label = extra_label.lower()
        extra_label = extra_label.encode('utf-8')
    return extra_label
```

Let's test this

```python
>>> def extract_extra_label(metadata_dict):
...     """Return the 'extra' label from metadata_dict"""
...     extra_label = metadata_dict.get('extra_label', None)
...     if extra_label:
...         extra_label = extra_label.lower()
...         extra_label = extra_label.encode('utf-8')
...     return extra_label
... 
>>> metadata_dict = {'extra_label': 'yohoo', 'foo': 'bar'}
>>> extract_extra_label(metadata_dict)
'yohoo'
>>> metadata_dict = {'foo': 'bar'}
>>> extract_extra_label(metadata_dict)
>>> 
```

Reply:
yep, thanks.